lab-integration/src/main.ts

```diff
@@ -0,0 +1,85 @@
+import { WatchEventType, promises as fs, watch } from "fs";
+import * as lockfile from "proper-lockfile";
+import path from "path";
+import { queue } from "async";
+import puppeteer from "puppeteer";
+
+import { DateTime } from "luxon";
+import { TestsResult } from "./types";
+
+const filePath = path.join(process.cwd(), "../test-results.json");
+
+const machines = new Map<string, string>([
+  ["5534", "1 - Prensa PRE-01"],
+  ["5533", "2 - Prensa PRE-02"],
+  ["5532", "3 - Prensa PRE-03"],
+]);
+
+(async () => {
+  console.info("SCRIPT IS RUNNING...");
+
+  let lastTestResultDate = DateTime.now();
+
+  const browser = await puppeteer.launch({ headless: false, slowMo: 100 });
+  const page = await browser.newPage();
+  await page.setViewport({ width: 720, height: 720 });
+  await page.goto("http://localhost:3000/submission");
```

Comment:
Let's initialize the `browser` here, but let's initialize `page` after we find new test results.